string file_name

---
string result
